LocaleUtils/index.js

```diff
@@ -0,0 +1,21 @@
+
+export async function assertNoEnglishLeaks(mockFile, viewText, noTranslationContent) {
```

Comment:
Shall we move code under `packages` folder?

Reply:
Can do as a follow up https://oktainc.atlassian.net/browse/OKTA-385962